tests/test_models_circuit_noise.py

```diff
@@ -15,7 +20,7 @@ def test_pauli_noise_channel(backend):
     c.add(gates.H(1))
     c.add(gates.PauliNoiseChannel(0, list(zip(["X", "Z"], [0.5, 0.3]))))
     c.add(gates.PauliNoiseChannel(1, list(zip(["Y", "Z"], [0.1, 0.3]))))
-    final_rho = backend.execute_circuit(c)
+    final_rho = backend.execute_circuit(c)._state
```

Comment:
Same as above
```suggestion
    final_rho = backend.execute_circuit(c)
```

I think in the cases where you use `backend.assert_allclose`, you don't need to call `.state()` explicitly because this is done automatically in the backend method.